src/legacy/ui/public/agg_types/buckets/terms.js

```diff
@@ -174,24 +149,11 @@ export const termsBucketAgg = new BucketAggType({
           const orderBy = params.orderBy;
           const paramDef = agg.type.params.byName.orderAgg;
 
-          // setup the initial value of orderBy
-          if (!orderBy && prevOrderBy === INIT) {
```

Comment:
This logic was moved to `order_agg.tsx` `useEffect` function (initialization step)